README.md

```
@@ -85,23 +116,22 @@ Each successful run attaches one self-contained package for every target to a
-```powershell
-pacman -S mingw-w64-ucrt-x86_64-gcc
+Extract a package into an install prefix (for example `~/.local`) so the
+executable lives in `bin` and its libraries live in `lib/little`. The CLI
+searches that installed native-library directory automatically. It also
+recognizes the older portable `libs/` directory beside the executable. A
+packaged library can be loaded by name, for example `loadLibrary("json")` or
+`loadLibrary("webui")`: the loader tries both `name.*` and `name/name.*`.
 
-gcc main.c src/little_buffer.c src/little.c src/little_common.c src/little_std.c src/little_loadlib.c src/little_std_io.c src/little_std_math.c src/little_std_array.c src/little_std_table.c src/little_std_string.c src/little_std_gc.c src/little_async.c -Wl,--export-all-symbols -o little
-```
+The supported build entry point is `build.ps1` (normally through `task build`).
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)